Game/Source/GDKTestGyms/UptimeCrossServerBeacon.h

```diff
@@ -0,0 +1,42 @@
+// Copyright (c) Improbable Worlds Ltd, All Rights Reserved
+
+#pragma once
+
+#include "NFRConstants.h"
+#include "GameFramework/Character.h"
+#include "UptimeCrossServerBeacon.generated.h"
+
+DECLARE_LOG_CATEGORY_EXTERN(LogUptimeCrossServerBeacon, Log, All);
+
+UCLASS()
+class AUptimeCrossServerBeacon :
+	public AActor
```

Comment:
`class AUptimeCrossServerBeacon : public AActor`
I'd like to put these in the same line instead.